### PR TITLE
Edits to SDSQuickStart

### PR DESCRIPTION
**Pull request by osimarkrogers**
Active voice  and task isolation

### CODE REVIEW

File: V1/Overview/SDSQuickStart.md [hunk 1]
```diff
@@ -35,20 +43,22 @@ Create an SDS type that describes the format of the data to be stored in a conta
...
-2. Run the following curl script:
+2. Save the JSON as a file with the name SDSCreateType.json.
+3. Run the following curl script:
 
 ```bash
 curl -i -d "@SDSCreateType.json" -H "Content-Type: application/json"  -X POST http://localhost:5590/api/v1/tenants/default/namespaces/default/types/Simple
 ```
 
-When this command completes successfully, an SDS type with the same name will have been created on the server. You can create any number of containers from the type, as long as they use a timestamp as an index and a 32 bit floating point value. You only need to do type creation the first time you send using a custom application. It does not cause an error if you resend the same definition at a later time.
+When the above command completes successfully, an SDS type with the same name is created on the server. You can create any number of containers from a single type, as long as they use a timestamp as an index and a 32 bit floating point value. You only need to create a type the first time you send data with a custom application. It does not cause an error to resend the same definition at a later time.
```
Comment (TinaUll) [inline V1/Overview/SDSQuickStart.md]: Can we not use words like "above" and "below"? Maybe "previous"

File: V1/Overview/SDSQuickStart.md [hunk 2]
```diff
@@ -58,20 +68,22 @@ Create a stream. As with an SDS type, you only need to do this once before sendi
...
-2. Run the following curl script:
+2. Save the JSON as a file with the name SDSCreateStream.json.
+3. Run the following curl script:
 
 ```bash
 curl -i -d "@SDSCreateStream.json" -H "Content-Type: application/json"  -X POST http://localhost:5590/api/v1/tenants/default/namespaces/default/streams/Simple
 ```
 
-When this command completes successfully, an SDS stream will have been created to store data defined by the type.
+When the above command completes successfully, an SDS stream is created on the server to store data defined by the specified type.
```
Comment (TinaUll) [inline V1/Overview/SDSQuickStart.md]: Same comment, maybe "previous"

File: V1/Overview/SDSQuickStart.md (hunk 1 above)
Comment (osimarkrogers) [reply V1/Overview/SDSQuickStart.md]: Good catch.  Fixed in branch.

File: V1/Overview/SDSQuickStart.md (hunk 2 above)
Comment (osimarkrogers) [reply V1/Overview/SDSQuickStart.md]: Fixed in branch.

**Pull request merged**